src/backend/base/langflow/services/database/models/flow/model.py

```diff
@@ -111,7 +113,8 @@ def validate_icon_atr(cls, v):
         return v
 
     @field_validator("data")
-    def validate_json(v):
+    @classmethod
+    def validate_json(cls, v):
```

Comment:
Or could be a `staticmethod`. Using a class method for consistency with pydantic's docs.